readers/influxdb/messages.go

```diff
@@ -239,8 +239,13 @@ func parseSenml(names []string, fields []interface{}) interface{} {
 				msgField.SetString(s)
 			}
 		case float64:
+			value, ok := fields[i].(string)
```

Comment:
`value` is not the best name for this variable. Maybe `f`, or `fs` (like field-string`) or something else would be more go-idiomatic